src/migrations/20230915094540-dependent-flags.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+          ALTER TABLE feature_strategies ADD COLUMN IF NOT EXISTS dependencies JSONB DEFAULT '[]'::jsonb NOT NULL;
+
+          CREATE VIEW features_view AS
+          SELECT
+              features.name as name,
+              features.description as description,
+              features.type as type,
+              features.project as project,
+              features.stale as stale,
+              features.impression_data as impression_data,
+              features.created_at as created_at,
+              features.archived_at as archived_at,
+              features.last_seen_at as last_seen_at,
+              feature_environments.last_seen_at as env_last_seen_at,
+              feature_environments.enabled as enabled,
+              feature_environments.environment as environment,
+              feature_environments.variants as variants,
+              environments.name as environment_name,
+              environments.type as environment_type,
+              environments.sort_order as environment_sort_order,
+              feature_strategies.id as strategy_id,
+              feature_strategies.strategy_name as strategy_name,
+              feature_strategies.parameters as parameters,
+              feature_strategies.constraints as constraints,
+              feature_strategies.sort_order as sort_order,
+              fss.segment_id as segments,
+              feature_strategies.title as strategy_title,
+              feature_strategies.disabled as strategy_disabled,
+              feature_strategies.variants as strategy_variants,
+              feature_strategies.dependencies as dependencies,
```

Comment:
this line is added all the other code copied from: 20230802141830-add-feature-and-environment-last-seen-at-to-features-view